tests/foreman/ui/test_settings.py

```diff
@@ -175,6 +175,60 @@ def test_negative_validate_error_message(session, set_original_property_value):
                 assert is_valid_error_message(str(context.value))
 
 
+@tier2
+def test_positive_selectors(session):
+    """"Testing input for selectors: dropdown, text area, input box
+
+    :id: 529ddd3a-1271-4043-9006-eac436b08b11
+
+    :expectedresults: Successfully add value to different selectors
+
+    :CaseImportance: High
+    """
+    uuid_input = f'[ {gen_string("alpha")} ]'
+    random_input = gen_string('alpha')
+    uuid_duplicate_prop = 'host_dmi_uuid_duplicates'
+    reg_host_prop = 'register_hostname_fact'
+    dep_solve_prop = 'content_view_solve_dependencies'
+    with session:
```

Comment:
instead of creating running same commands again I would do in this way

```
In [6]: dep_solve_prop = ('content_view_solve_dependencies', 'Yes')                                                                     

In [7]: reg_host_prop = ('register_hostname_fact', 'random_input')                                                                      

In [8]: for setting_attr, value in [dep_solve_prop, reg_host_prop]: 
   ...:     print(setting_attr, value) 
   ...:                                                                                                                                 
content_view_solve_dependencies Yes
register_hostname_fact random_input

```